plugins/py3embed/CMakeLists.txt

```diff
@@ -71,6 +71,10 @@ target_link_libraries(
     roxiemem
     jlib)
 
+if (LINUX)
```

Comment:
What is this for?

Reply:
It is to resolve some import errors:
````
/usr/bin/ld: ../../vcpkg_installed/x64-linux/debug/lib/libpython3.10d.a(posixmodule.o): in function `os_openpty_impl':
/home/runner/work/vcpkg/vcpkg/buildtrees/python3/x64-linux-dbg/./../src/v3.10.1-8b5dfc5615.clean/Modules/posixmodule.c:7247: undefined reference to `openpty'
/usr/bin/ld: ../../vcpkg_installed/x64-linux/debug/lib/libpython3.10d.a(posixmodule.o): in function `os_forkpty_impl':
/home/runner/work/vcpkg/vcpkg/buildtrees/python3/x64-linux-dbg/./../src/v3.10.1-8b5dfc5615.clean/Modules/posixmodule.c:7348: undefined reference to `forkpty'
```

Reply:
And looks like it didn't work on the GH build machine...